handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -474,21 +473,26 @@ public synchronized SSLEngineResult unwrap(
         }
 
         // Throw requried runtime exceptions
-        if (src == null) {
-            throw new NullPointerException("src");
+        if (srcs == null) {
+            throw new NullPointerException("srcs");
+        }
+        if (srcsOffset >= srcs.length
+                || srcsOffset + srcsLength > srcs.length) {
+            throw new IndexOutOfBoundsException(
+                "offset: " + srcsOffset + ", length: " + srcsLength +
+                        " (expected: offset <= offset + length <= srcs.length (" + srcs.length + "))");
```

Comment:
```
"offset: " + srcsOffset + ", length: " + srcsLength +
        " (expected: offset <= offset + length <= srcs.length (" + srcs.length + "))");
```
could be:
```
"offset: " + srcsOffset + ", length: " + srcsLength +
" (expected: offset <= offset + length <= srcs.length (" + srcs.length + "))");
```
according to our updated style. Same for other multi-line string concats.